### DISCUSSION TITLE
How to force `Auto fill` of case class in tests

### DISCUSSION THREAD

**Discussion by bartoszb-rgb** [scalameta/nvim-metals#595] [Q&A]
Hey,
Auto fill works perfect when used in within single build target.
When I create test (next build target) I would expect it also to work... For some reason it is not working....
Any idea how to fix it?

**Comment by ckipp01** [comment]
Hey @bartoszb-rgb, you'll have to give me some more context here. What exactly do you mean by "auto fill of case class". Do you mean auto completion? Doe you mean auto-filling named parameters? Also can you give me some more details about what build tool you're using and by build target, are you just creating an entirely new build target and then trying to complete it in? Is that build target registered? Did you look at the Doctor to ensure that it sees it there?

**Reply by bartoszb-rgb** [reply]
Yes, I mean auto-filling named parameters.
Well project is set on sbt and uses Scala 2.13.11
Metals (v1.0.0) on the other hand is set on Bloop (v1.5.8)
When I run Doctor it shows like:
```
### <project_name> // all fine
### <project_name>-it // all fine
### <project_name>-test // all fine
### <project_name>-build // no definitions and debugging
### <project_name>-build-build // no definitions and debugging
```

Behavior on `src` is correct, it auto generates all fields, but it feature is not accessible for test files in `test` or `it`.|

**Reply by ckipp01** [reply]
Hi @bartoszb-rgb I just tested this in a minimal project and it seemed to work fine. I got the completions in the same target as where I created it, and then moved to the test target and also got them there. You'll have to provide some sort of minimal reproduction of your issue.
